### PR TITLE
Add support for element labels #142519949

### PR DESCRIPTION
**Pull request by jfischoff**
Fulfills https://www.pivotaltracker.com/story/show/142519949

This commit adds element labels support. It copies a modifies the code and tests for sensor labels. 

Some existing tests were updated due to different data.

### CODE REVIEW

**Comment by lseppala** [comment]
How about adding methods to add/replace/remove labels for a given element? We support this symmetry with sensors, (i.e., you can modify the sensors in a label or the labels of a sensor). For completeness in the public client, we should do this for elements as well.

File: spec/helium/label_spec.rb [hunk 1]
```diff
@@ -29,21 +29,191 @@
...
+  end
+
+  context 'with mulitple sensors' do
+    use_cassette 'labels/remove_elements_multiple'
+
+    it "removes elements from a label without overriding existing relationships" do
+      # create a new label
+      new_label = client.create_label(name: "A Test Label")
+
+      # create some sensors
```
Comment (allenan) [inline spec/helium/label_spec.rb]: comments need updating in this spec

**Comment by jfischoff** [comment]
Capital idea. On it.

File: spec/helium/label_spec.rb (hunk 1 above)
Comment (jfischoff) [inline spec/helium/label_spec.rb]: ACK

**Comment by allenan** [comment]
+1 looks good to me

**Comment by lseppala** [comment]
+1 from me too, merging

**Pull request merged**